queries/useConversationMessages.ts

```diff
@@ -100,21 +162,23 @@ export const addConversationMessage = (
     setConversationMessages(account, topic, {
       ids: [message.id],
       byId: { [message.id]: message },
+      reactions: {},
     });
     return;
   }
   previousMessages.byId[message.id] = message;
   setConversationMessages(account, topic, {
     ids: [message.id, ...previousMessages.ids],
     byId: previousMessages.byId,
+    reactions: previousMessages.reactions,
   });
 };
 
-export const prefetchConversationMessages = (
+export const prefetchConversationMessages = async (
```

Comment:
We should have a function like `getConversationMessagesQueryOptions` and reuse that instead everywhere. The `react-query` maintainer recommend this method. https://tkdodo.eu/blog/the-query-options-api